assets/shaders/extended_material_bindless.wgsl

```diff
@@ -0,0 +1,77 @@
+// The shader that goes with `extended_material_bindless.rs`.
+
+#import bevy_pbr::{
+    forward_io::{FragmentOutput, VertexOutput},
+    mesh_bindings::mesh,
+    pbr_bindings::{material_array, material_indices},
+    pbr_fragment::pbr_input_from_standard_material,
+    pbr_functions::{apply_pbr_lighting, main_pass_post_lighting_processing},
+}
+#import bevy_render::bindless::{bindless_samplers_filtering, bindless_textures_2d}
+
+// Stores the indices of the bindless resources in the bindless resource arrays,
+// for the `ExampleBindlessExtension` fields.
+struct ExampleBindlessExtendedMaterialIndices {
```

Comment:
ideally this shader would show the non-bindless fallback as well.

Reply:
Done.